Makefile.in

```
@@ -26,8 +26,8 @@ TAG ?= $(shell git rev-parse --short HEAD)
 
 ES_IMAGE ?= docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.0
 
-SW_OAP_IMAGE ?= ghcr.io/apache/skywalking/oap:74fbc3d8ecce103eeefc2a7e52b3d369b0b2c956
-SW_ROCKET_BOT_IMAGE ?= ghcr.io/apache/skywalking/ui:74fbc3d8ecce103eeefc2a7e52b3d369b0b2c956
+SW_OAP_IMAGE ?= ghcr.io/apache/skywalking/oap:b670ea6031d0e23c8947459ccb9c02ee435ca9f5
+SW_ROCKET_BOT_IMAGE ?= ghcr.io/apache/skywalking/ui:b670ea6031d0e23c8947459ccb9c02ee435ca9f5
```

```suggestion
SW_OAP_IMAGE ?= ghcr.io/apache/skywalking/oap:4e750e69fbe34a9d6e87b4db7dda955a335736db
SW_ROCKET_BOT_IMAGE ?= ghcr.io/apache/skywalking/ui:4e750e69fbe34a9d6e87b4db7dda955a335736db
```

If e2e could pass, I will update this to latest again.